### PR TITLE
Implement actuator support for System

### PR DESCRIPTION
**Pull request by tjstienstra**
#### Brief description of what is fixed or changed
This PR implement support for the new actuator class in `System`.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/physics/mechanics/_system.py [hunk 1]
```diff
@@ -347,6 +349,19 @@ def loads(self, loads):
         loads = self._objects_to_list(loads)
         self._loads = [_parse_load(load) for load in loads]
 
+    @property
+    def actuators(self):
+        """Tuple of actuators that have are in the system."""
```
Comment (brocksam) [inline sympy/physics/mechanics/_system.py:354]: Typo

File: sympy/physics/mechanics/_system.py [hunk 2]
```diff
@@ -656,6 +671,22 @@ def apply_gravity(self, acceleration):
...
+
+        Parameters
+        ==========
+
+        *actuators : subclass of ActuatorBase
+            One or more actuators.
+
+        """
+        self._check_objects(actuators, self.actuators, ActuatorBase,
+                            'Actuators',
```
Comment (brocksam) [inline sympy/physics/mechanics/_system.py:686]: Formatting? And date I ask about why this method needs two strings of the same thing in different capitalisation cases?

File: sympy/physics/mechanics/tests/test_system_class.py [hunk 3]
```diff
@@ -23,7 +24,7 @@ def _empty_system_setup(self):
     def _empty_system_check(self, exclude=()):
         matrices = ('q_ind', 'q_dep', 'q', 'u_ind', 'u_dep', 'u', 'kdes',
                     'holonomic_constraints', 'nonholonomic_constraints')
-        tuples = ('loads', 'bodies', 'joints')
+        tuples = ('loads', 'bodies', 'joints', "actuators")
```
Comment (brocksam) [inline sympy/physics/mechanics/tests/test_system_class.py:27]: Mix of quote types.

File: sympy/physics/mechanics/tests/test_system_class.py [hunk 4]
```diff
@@ -206,6 +207,8 @@ def test_speeds(self, _empty_system_setup, args, kwargs, exp_u_ind,
          (u[0] - u[1],), {}),
         ('bodies', 'add_bodies', (RigidBody('body'),), {}),
         ('loads', 'add_loads', (Force(Point('P'), ReferenceFrame('N').x),), {}),
+        ('actuators', 'add_actuators', (TorqueActuator(
+            symbols("T"), ReferenceFrame('N').x, ReferenceFrame("A")),), {}),
```
Comment (brocksam) [inline sympy/physics/mechanics/tests/test_system_class.py:211]: Mix of quote types.

File: sympy/physics/mechanics/tests/test_system_class.py [hunk 5]
```diff
@@ -582,15 +600,16 @@ def test_cart_pendulum_kanes(self):
...
         system.form_eoms()
         assert isinstance(system.eom_method, KanesMethod)
         assert (_simplify_matrix(system.mass_matrix - ImmutableMatrix(
             [[mp + mc, mp * l * cos(qp)], [mp * l * cos(qp), mp * l ** 2]]))
                 == zeros(2, 2))
         assert (_simplify_matrix(system.forcing - ImmutableMatrix([
-            [mp * l * up ** 2 * sin(qp) + F], [-mp * g * l * sin(qp)]]))
-                == zeros(2, 1))
+            [mp * l * up ** 2 * sin(qp) + F], [-mp * g * l * sin(qp) + k * qp]])
+                                 ) == zeros(2, 1))
```
Comment (brocksam) [inline sympy/physics/mechanics/tests/test_system_class.py:612]: Why this formatting?

**Review by brocksam — COMMENTED** [review]
Minor review comments. But generally looks good.

File: sympy/physics/mechanics/_system.py (hunk 2 above)
Comment (TJStienstra) [reply sympy/physics/mechanics/_system.py:686]: Sometimes they are actually more different, namely in the case of constraints. There the object name (e.g. holonomic constraints) and the type name (e.g. expressions) is not the same.

Comment (brocksam) [reply sympy/physics/mechanics/_system.py:686]: Thanks.

**Comment by brocksam** [comment]
Looks like we might be at the whim of the SymPy bot again here for a bit...

**Review by brocksam — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**